.rubocop.yml

```diff
@@ -2,7 +2,4 @@ inherit_from:
   - http://shopify.github.io/ruby-style-guide/rubocop.yml
 
 AllCops:
-  TargetRubyVersion: 2.1
-
-Layout/IndentHeredoc:
-  EnforcedStyle: active_support
+  TargetRubyVersion: 2.3
```

Comment:
> The new # frozen_string_literal: true came from running bx rubocop -a. Dunno why it didn't care about those files until now.

Because it has been validating code for 2.1, and in that version there was no such thing as frozen string literal